modules/client-configuration/pages/supported-features-ubuntu.adoc

```diff
@@ -0,0 +1,69 @@
+[[supported-features-ubuntu]]
+= Supported Ubuntu Features
+
+
+This table lists the availability of various features on {ubuntu} clients.
+
+[NOTE]
+====
+ Debian is not currently supported by {suse}.
+====
+
+The icons in this table indicate:
+
+* icon:check[role="green"] the feature is available on both traditional and Salt clients
+* icon:times[role="danger"] the feature is not available
+* icon:question[role="gray"] the feature is under consideration, and may or may not be made available at a later date
+* ``Traditional`` the feature is supported only on traditional clients
+* ``Salt`` the feature is supported only on Salt clients.
+
+
+[cols="1,1,1", options="header"]
+.Supported Features on {ubuntu} Operating Systems
+|===
+| Feature | {ubuntu}{nbsp}16.04 | {ubuntu}{nbsp}18.04
+| Client  | icon:check[role="green"] | icon:check[role="green"]
+| Registration | ``Salt`` | ``Salt``
+| Install packages | ``Salt`` | ``Salt``
+| Apply patches | icon:times[role="danger"] | icon:times[role="danger"]
+| Remote commands | ``Salt`` | ``Salt``
+| System package states | ``Salt`` | ``Salt``
+| System custom states | ``Salt`` | ``Salt``
+| Group custom states | ``Salt`` | ``Salt``
+| Organization custom states    | ``Salt`` | ``Salt``
+| System set manager (SSM) | ``Salt`` | ``Salt``
+| Service pack migration | icon:times[role="danger"] | icon:times[role="danger"]
+| Basic Virtual Guest Management icon:asterisk | icon:check[role="green"] | icon:check[role="green"]
+| Advanced Virtual Guest Management icon:asterisk | ``Salt`` | ``Salt``
+| Virtual Guest Installation ({ay}), as Host OS | icon:times[role="danger"] | icon:times[role="danger"]
```

Comment:
```suggestion
| Virtual Guest Installation (Kickstart), as Host OS | icon:times[role="danger"] | icon:times[role="danger"]
```

If we can support it at some point in time, it will be kickstart.